authserver/app/app.py

```diff
@@ -79,6 +80,23 @@ def after_request(response):
             logging.info(jsonstr)
         return response
 
+    def handle_errors(e):
+        print(f"""{e}, app.py, line 83""")
```

Comment:
I suggest replacing the `print` with `logging`. (I think the print statements in the DTM API are left over from the neo4j connectivity debugging extravaganza.)